Frends.Community.PowerShell.Tests/PowerShellTests.cs

```diff
@@ -124,7 +127,7 @@ public void RunScript_ShouldRunScriptFromParameter()
                 ReadFromFile = false,
```

Comment:
https://github.com/CommunityHiQ/Frends.Community.PowerShell/blob/a0b1eb70068bbf2bb166cb1532f44306342d7468/Frends.Community.PowerShell.Tests/PowerShellTests.cs#L116
Please use `AreEqual` instead of `That`.